model/project_vars.go

```diff
@@ -206,32 +206,24 @@ func (projectVars *ProjectVars) FindAndModify(varsToDelete []string) (*adb.Chang
 func (projectVars *ProjectVars) GetVars(t *task.Task) map[string]string {
 	vars := map[string]string{}
 	for k, v := range projectVars.Vars {
-		if !projectVars.AdminOnlyVars[k] {
+		if !projectVars.AdminOnlyVars[k] || projectVars.ShouldGetAdminOnlyVars(t) {
```

Comment:
We should store ShouldgetAdminOnlyVars as a variable so we aren't re-calculating it on each iteration of the if statement

Reply:
done